lib/prometheus_exporter/middleware.rb

```diff
@@ -44,14 +44,24 @@ def call(env)
       controller = params["controller"]
     end
 
-    @client.send_json(
+    obj = {
       type: "web",
       timings: info,
       queue_time: queue_time,
       action: action,
       controller: controller,
       status: status
-    )
+    }
+    labels = custom_labels(env)
+    if labels.present? && labels.length > 0
+      obj = obj.merge(custom_labels: labels)
+    end
+    
+    @client.send_json(obj)
+  end
+
+  def custom_labels(env)
+    {}
```

Comment:
should this do something other than return an empty object?

Reply:
what do you mean? this shouldn't change any current behavior in the library besides make it easy for subclasses to provide custom_labels

Reply:
ahh gotcha. Sorry, confused by languages without explicit interfaces..

Reply:
Might be worth a comment or something detailing that this method exists largely to be overridden by its subclass